instrumentation/ratpack-1.4/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/ratpack/ActionWrapper.java

```diff
@@ -35,7 +35,11 @@ public void execute(T t) throws Exception {
     if (delegate instanceof ActionWrapper) {
       return delegate;
     }
+    Context context = Context.current();
+    if (context == Context.root()) {
```

Comment:
Can you add some comment to explain why wrapping actions depend on the context being root or not?